core/src/raw/http_util/header.rs

```diff
@@ -186,6 +196,10 @@ pub fn parse_into_metadata(path: &str, headers: &HeaderMap) -> Result<Metadata>
         m.set_content_disposition(v);
     }
 
+    if let Some(v) = parse_version_id(headers)? {
```

Comment:
Hi, please implement those logic in oss instead of `raw`.

Reply:
👌